tests/tiling/tools/freemem.c

```diff
@@ -0,0 +1,33 @@
+#include <stdio.h>
```

Comment:
Maybe add a comment at the top of this file (and the other files regarding the CX2 memory issue) to make it clear to users that this is not really part of the tiling feature but a workaround for a local issue.